grpc/bcodes.go

```diff
@@ -21,13 +20,12 @@ import (
 // TODO(#2507): Deprecated, remove once boulder/errors code is deployed
 const (
 	MalformedRequestError = iota + 100
-	NotSupportedError
+	_
 	UnauthorizedError
 	NotFoundError
-	LengthRequiredError
+	__
 	RateLimitedError
-	BadNonceError
-	NoSuchRegistrationError
+	___
```

Comment:
Is this increasing `_`, `__`, `___` sequence a visual style choice or syntax? I haven't had to skip more than one enum value before!

Reply:
It was because I figured `_` would become a real name and I'd have to avoid duplicates. I never bother to go back and check whether I could use a single `_` everywhere. Turns out I can!